file_store/src/verified_mapping_event.rs

```diff
@@ -0,0 +1,63 @@
+use crate::{
+    traits::{MsgDecode, MsgTimestamp, TimestampDecode, TimestampEncode},
+    Error, Result,
+};
+use chrono::{DateTime, Utc};
+use helium_proto::services::poc_mobile::VerifiedSubscriberMappingEventV1;
+use serde::{Deserialize, Serialize};
+use sqlx::Row;
+
+#[derive(Clone, Deserialize, Serialize, Debug, PartialEq)]
+pub struct VerifiedSubscriberMappingEvent {
+    pub subscriber_id: Vec<u8>,
+    pub total_reward_points: u64,
+    pub timestamp: DateTime<Utc>,
```

Comment:
for completeness & transparency you should carry the pubkey through to this msg.  That fits with existing convention of ingestor received requests where the only field dropped tends to be the signature